lib/http_perform.c

```diff
@@ -454,8 +454,8 @@ sf_bool STDCALL http_perform(CURL *curl,
 
         // When renew timeout is reached, stop retry and return to the caller
         // to renew request
-        if ((retry) && (renew_timeout > 0) &&
-            ((time(NULL) - elapsedRetryTime) >= renew_timeout)) {
+        if (((retry) && (renew_timeout > 0) &&
+            ((time(NULL) - elapsedRetryTime) >= renew_timeout)) || renew_timeout < 0) {
```

Comment:
This is somehow misleading, please put `renew_timeout < 0` in a new line and put others in the same line.

Reply:
I applied it @sfc-gh-jszczerbinski. Thank you for your suggestion.